# AcommsCycleStatus
# - Used to communicate status from ROS -> MOOS
Header header
uint16 pings_attempted
uint16 pings_successful
float32 pings_pct_successful
float32 min_range
float32 max_range
time last_range_timestamp
duration last_range_interval
uint8 last_range_target
float32 last_range_distance